yarn.lock

```diff
@@ -7962,6 +7975,15 @@ rc@^1.0.1, rc@^1.1.6, rc@^1.2.1, rc@^1.2.7:
     minimist "^1.2.0"
     strip-json-comments "~2.0.1"
 
+react-autosize-textarea@^7.0.0:
```

Comment:
@marcdumais-work Could you check again please our deps? 🙏

Reply:
Done - I did the thorough process that includes recursive production dependencies, and all dependencies look good license-wise!

Reply:
I wonder how do you use ClearlyDefined UI, there are so many records. Do you scroll to find new deps or you use some filters?

Reply:
I do not really pay attention about new vs old dependencies; it takes a few minutes to go through them all, regardless. 

I sort by license, then I can go quickly with "page down", looking at the licenses for all packages. e.g. there are many many pages of MIT, so you can go though those really quick.